tst/App.spec.tsx

```diff
@@ -0,0 +1,10 @@
+import { default as App, default as Database } from "../App";
+
+jest.mock("../src/database/parse-server");
```

Comment:
This is to make sure we don't actually call the real implementation of `Parse.initialize(...)` in the tests

Reply:
this is perfect!